### ISSUE TITLE
OrderedFixtureInterface and FixtureInterface

### ISSUE THREAD

**Issue by J7mbo** [doctrine/data-fixtures#174]
This might be a weird request, but I find having to implement both `OrderedFixtureInterface` and `FixtureInterface` in order to use fixtures (doctrine doesn't seem to pick up a fixture unless it implements the latter) weird in itself.

I propose `interface OrderedFixtureInterface extends FixtureInterface { /** SNIP **/ }`. This means that:
- Either of the fixture interfaces require the load method (because, well, they're both fixtures so they both should require this method to be implemented anyway)
- You can implement `FixtureInterface` if you don't care about order, **or** `OrderedFixtureInterface` if you care about order.
- Doctrine will pick up either one as a fixture when attempting to load them

Basically a really simple change, but from a design perspective it makes sense... I decided to take the time to suggest such a minor change. What's your opinion on this? Good idea? (If you think the idea is okay I would happily provide a PR & any tests).

**Comment by lavoiesl** [comment]
Since `OrderedFixtureInterface` and `DependentFixtureInterface` are useless without being `FixtureInterface`, I would tend to agree on this. I tried searching in the original [PR’s history](https://github.com/doctrine/data-fixtures/pull/5/files#diff-1e4e94abac5fa4ed139437d23ae450e8), and nothing is mentioned.

However, `Loader::isTransient` specifically tests for `FixtureInterface`, are `OrderedFixtureInterface` useful without `FixtureInterface` ?
